samples/trace-utility-network/src/main/java/com/esri/arcgismaps/sample/traceutilitynetwork/screens/TraceUtilityNetworkScreen.kt

```diff
@@ -90,6 +90,9 @@ fun TraceUtilityNetworkScreen(sampleName: String) {
     // Create the view model used by the sample
     val mapViewModel: TraceUtilityNetworkViewModel = viewModel()
 
+    // Load map, utility network, and adds layers. 
+    LaunchedEffect(Unit) { mapViewModel.initializeTrace() }
```

Comment:
sorry to string you along. This will run every time the composition recomposes, but the viewmodel only needs to initialize once. Is there a way to check if the viewmodel has been initialized?

Reply:
Good point. Added a check in the `LaunchedEffect` to run if the network is not loaded.